client/css/_overrides.scss

```diff
@@ -136,9 +136,38 @@ blockquote {
 .card {
   $card-border-color: $grey-darker;
   box-shadow: 0 0 1px $grey-light;
+  min-height: calc(65vh);
```

Comment:
I don't think you need calc if you're not using an operators. `65vh) should be ok, right?